aws-codeartifact-domain/src/main/java/software/amazon/codeartifact/domain/Translator.java

```diff
@@ -88,10 +88,8 @@ static DescribeDomainRequest translateToReadRequest(
   static GetDomainPermissionsPolicyRequest translateToGetDomainPermissionPolicy(
       final ResourceModel model, final ResourceHandlerRequest<ResourceModel> request
   ) {
-    String accountId = request.getAwsAccountId();
     return GetDomainPermissionsPolicyRequest.builder()
         .domain(model.getDomainName())
-        .domainOwner(accountId)
         .build();
```

Comment:
kinda odd that we were first using the caller's accountId instead of just grabbing it from the model itself

and now we are just omitting it. I get that the caller should be the domain owner in this case. But if we decide that anyone can create an independent resource for a DomainPolicy, we might pass in the domain Owner to that method and that would then need to be translated accordingly